CONTRIBUTING.md

```diff
@@ -37,15 +38,7 @@ From a separate terminal, start running a flutter app to connect to DevTools:
 
 ## Development
```

Comment:
Can you please move the "Development" section above "Workflow for making changes"? This seems more natural to follow "Development Prep" instead of interrupting development-related instructions to talk about using Git.

Reply:
done